src/components/Visualisers/Map.tsx

```diff
@@ -0,0 +1,191 @@
+import { useCallback, FC, ReactElement, useEffect, useMemo, memo } from 'react';
+import ellipsisAddress from '../../utils/ellipsisAddress';
+import ReactFlow, {
+  Controls,
+  Background,
+  useNodesState,
+  useEdgesState,
+  addEdge,
+} from 'reactflow';
+// 👇 you need to import the reactflow styles
+import 'reactflow/dist/style.css';
+import { Network } from "../../redux/networks";
+import NextLink from 'next/link';
+import { sfSubgraph } from "../../redux/store";
+import { ethers } from 'ethers'
+import Block from './Blockies';
+import UserBlock from './UserBlock';
+import { Typography, Box } from '@mui/material';
+
+export enum StreamStatus {
+  Active,
+  Inactive,
+}
+
+interface edge {
+  id: string,
+  source: string,
+  target: string,
+  label?: string,
+  animated?: boolean
+}
+
+interface node {
+  id: string,
+  position: {x: number, y: number},
+  data: {label: ReactElement},
+  flowRate?: string,
+}
+
+const Map: FC<{
+  network: Network;
+  accountAddress: string;
+}> = ({network, accountAddress}): ReactElement => {
+
+  const initialNodes: node[] = [
+    {
+      id: accountAddress,
+      position: { x: 500, y: 300 },
+      data: {
+        label:
+        <UserBlock network={network!} address={accountAddress} />
+      }
+    },
+  ];
+
+  const initialEdges: edge[] = [{ id: 'e1-2', source: '1', target: '2' }];
+
+  const [nodes, setNodes, onNodesChange] = useNodesState(initialNodes);
+  const [edges, setEdges, onEdgesChange] = useEdgesState(initialEdges);
+
+  const onConnect = useCallback((params: any) => setEdges((eds) => addEdge(params, eds)), [setEdges]);
+
+  //query incoming and outgoing streams on user address
+  const outgoingStreams = sfSubgraph.useStreamsQuery({
+    chainId: network.chainId,
+    filter: {
+      sender: accountAddress
+    },
+    pagination: {
+      take: Infinity
+    }
+  });
+
+  const incomingStreams = sfSubgraph.useStreamsQuery({
+    chainId: network.chainId,
+    filter: {
+      receiver: accountAddress
+    },
+    pagination: {
+      take: Infinity
+    }
+  });
+
+  //Filter out 0 current flow rate streams
+  const OutgoingStreams: any = useMemo(() => (outgoingStreams.data?.data.filter((row) => row.currentFlowRate !== '0') || []), [outgoingStreams.data]);
```

Comment:
I think you can filter out inactive streams already in the useStreamsQuery with currentFlowRate_not: "0" filter.